addons/project/views/project_views.xml

```diff
@@ -209,6 +209,37 @@
             </field>
         </record>
 
+        <record id="project_view_kanban" model="ir.ui.view">
```

Comment:
@kic-odoo Why you add this view? the kanban view for project.project already exists.

Reply:
add view_id for open_view_project_all_config

Reply:
@adr-odoo ,
project.project existing kanban displya for dashboard and 
for configuration project kanban we need limited field (different from existing kanban.) 

second i didnot get your point: why we need to use view_id for open_view_project_all_config
it's already open list view (open_view_project_all_config)..

Reply:
on "open_view_project_all_config" the view mode is set to "tree,kanban,form".
But now we have two kanban views for the same model.
Maybe we want to precise wich one we want to open. I suppose the new one.

Reply:
@adr-odoo  by default the dashboard opens from config menu, i have created kanban view for project.project for config menu and defined the view_id for dashboard in open_view_project_all (dashboard action). now the new kanban view opens from this action open_view_project_all_config(by default list view),   so its working fine view_id here is not needed.